staging/src/k8s.io/client-go/tools/remotecommand/v1.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/golang/glog"
 	"k8s.io/apimachinery/pkg/util/httpstream"
-	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/client-go/pkg/api"
```

Comment:
It looks like it the api package is needed only for some constants. These constants should exist in the api/v1 package as well, could you use the copies there? Let's copy those constants to api/v1 if they are not already there